### PR TITLE
Blog: New Categories for Matrix Spec Changes

### PR DESCRIPTION
**Pull request by anoadragon453**
Categories are a thing! Blog about them!

### CODE REVIEW

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+The Spec Core Team itself is made up of 8 members, each of which have
+separate full-time jobs. 7 of these are employees of New Vector Ltd. All team
+members are well-placed to be on the team given their wide breadth of
+knowledge across the Matrix.org ecosystem, however the majority are some of
+the most busy pushing forward Riot, Synapse and Dendrite. This limits the
+amount of MSCs that the team can effectively work on at a given time.
+
+The team understands that there are MSCs that provide incredibly useful
+features, such as[ support for LaTeX in
+messages](https://github.com/matrix-org/matrix-doc/pull/2191) or[ the ability
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:43]:
```suggestion
messages](https://github.com/matrix-org/matrix-doc/pull/2191) or [the ability
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+to "knock" on rooms](https://github.com/matrix-org/matrix-doc/pull/2403), and
+would undoubtedly like to see them land at some point.
+
+But there is also a large backlog of MSCs that provide even more fundamental
+fixes and additions to the protocol that the team needs to prioritise. These
+include things like [cross-signing
+devices](https://github.com/matrix-org/matrix-doc/pull/2472),[ the
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:50]:
```suggestion
devices](https://github.com/matrix-org/matrix-doc/pull/2472), [the
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+Communities rewrite](https://github.com/matrix-org/matrix-doc/pull/1772) and
+finally merging[ reactions and
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:52]:
```suggestion
finally merging [reactions and
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  - General
+author: Andrew Morgan
+image: https://matrix.org/blog/img/2020-04-28-msc-banner.png
+---
+
+# New Categories for Matrix Spec Changes
+
+On April 14th, the Spec Core Team conducted a long-overdue retrospective
+about the things that were working in the[ Matrix Spec Proposal
+process](https://matrix.org/docs/spec/proposals), and those that were not.
```
Comment (benparsons) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:14]: line break will break the markdown parser

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+While we announce what MSCs we're focusing on during a given week during[
+TWIM](https://matrix.org/twim), it's not as clear which items we're looking
+to pull from the backlog next. To help tackle this, and to help keep us
+honest, we've begun putting each MSC into either "feature", "maintenance", or
+"core" buckets. This materialises in the form of github tags, which can be
+used to filter the list of MSCs like so:[
+feature](https://github.com/matrix-org/matrix-doc/issues?q=is%3Aopen+label%3Akind%3Afeature),[
+maintenance](https://github.com/matrix-org/matrix-doc/labels/kind%3Amaintenance),[
+core](https://github.com/matrix-org/matrix-doc/labels/kind%3Acore). For a
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:63]:
```suggestion
used to filter the list of MSCs like so:
[feature](https://github.com/matrix-org/matrix-doc/issues?q=is%3Aopen+label%3Akind%3Afeature),
[maintenance](https://github.com/matrix-org/matrix-doc/labels/kind%3Amaintenance),
[core](https://github.com/matrix-org/matrix-doc/labels/kind%3Acore). For a
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+
+## **Future**
+
+We'll try this approach out over the next few months and see how it goes. The
+next Spec Core Team retro will occur in the middle of May, where we will
+review the process once again.
+
+For now, if you have any feedback please come and chat with us in[
+#matrix-spec:matrix.org](https://matrix.to/#/#matrix-spec:matrix.org) :)
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:83]:
```suggestion
For now, if you have any feedback please come and chat with us in
[#matrix-spec:matrix.org](https://matrix.to/#/#matrix-spec:matrix.org) :)
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+general lack of activity from the Spec Core Team members on proposals that
+have not yet started a Final Comment Period.
+
+We deeply apologize for the frustration this has likely caused many MSC
+authors, and want to shed some light on the reasoning behind it, and what we
+plan to do to prevent leaving authors in the dark about why there may be no
+Spec Core Team activity on their proposal.
+
+
+## **Proposal Triaging**
```
Comment (benparsons) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:27]: `##` is already an h2, don't need to bold it

Comment (benparsons) [reply gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:27]: same for the other title

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+The Spec Core Team itself is made up of 8 members, each of which have
+separate full-time jobs. 7 of these are employees of New Vector Ltd. All team
+members are well-placed to be on the team given their wide breadth of
+knowledge across the Matrix.org ecosystem, however the majority are some of
+the most busy pushing forward Riot, Synapse and Dendrite. This limits the
+amount of MSCs that the team can effectively work on at a given time.
+
+The team understands that there are MSCs that provide incredibly useful
+features, such as[ support for LaTeX in
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:42]:
```suggestion
features, such as [support for LaTeX in
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+categories:
+  - General
+author: Andrew Morgan
+image: https://matrix.org/blog/img/2020-04-28-msc-banner.png
+---
+
+# New Categories for Matrix Spec Changes
+
+On April 14th, the Spec Core Team conducted a long-overdue retrospective
+about the things that were working in the[ Matrix Spec Proposal
```
Comment (turt2live) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:13]:
```suggestion
about the things that were working in the [Matrix Spec Proposal
```

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx (hunk 8 above)
Comment (benparsons) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:42]: line break

Comment (benparsons) [reply gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:42]: more below

**Comment by anoadragon453** [comment]
Curse you Doc to Markdown!

**Comment by anoadragon453** [comment]
Hrm sad.

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 10]
```diff
@@ -0,0 +1,85 @@
+---
+date: '2020-04-28'
+title: New Categories for Matrix Spec Changes
+categories:
+  - General
+author: Andrew Morgan
+image: https://matrix.org/blog/img/2020-04-28-msc-banner.png
+---
+
+# New Categories for Matrix Spec Changes
```
Comment (benparsons) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:10]: title is not needed, it's injected by the template, this line should be removed

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+While we announce what MSCs we're focusing on during a given week during
+[TWIM](https://matrix.org/twim), it's not as clear which items we're looking
+to pull from the backlog next. To help tackle this, and to help keep us
+honest, we've begun putting each MSC into either "feature", "maintenance", or
+"core" buckets. This materialises in the form of github tags, which can be
+used to filter the list of MSCs like so:
+[feature](https://github.com/matrix-org/matrix-doc/issues?q=is%3Aopen+label%3Akind%3Afeature),
+[maintenance](https://github.com/matrix-org/matrix-doc/labels/kind%3Amaintenance),
+[core](https://github.com/matrix-org/matrix-doc/labels/kind%3Acore). For a
```
Comment (benparsons) [inline gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:65]: format on these three links is not consistent - does it matter?

**Comment by benparsons** [comment]
one more thing, can the filename be changed to match the date

File: gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx (hunk 11 above)
Comment (anoadragon453) [reply gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:65]: Ah, probably.

Comment (anoadragon453) [reply gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx:65]: oh, wait what do you mean?

Comment (benparsons) [reply gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx]: I mean the links themselves

https://github.com/matrix-org/matrix-doc/issues?q=is%3Aopen+label%3Akind%3Afeature
https://github.com/matrix-org/matrix-doc/labels/kind%3Amaintenance
https://github.com/matrix-org/matrix-doc/labels/kind%3Acore

May not matter!

Comment (anoadragon453) [reply gatsby/content/blog/2020/04/2020-04-22-msc-categories.mdx]: Ahh, yes those should probably be consistent.

File: gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+author: Andrew Morgan
+image: https://matrix.org/blog/img/2020-04-28-msc-banner.png
+---
+
+On April 14th, the Spec Core Team conducted a long-overdue retrospective
+about the things that were working in the
+[Matrix Spec Proposal process](https://matrix.org/docs/spec/proposals),
+and those that were not.
+
+The most glaring item on the list was the very sluggish pace that most Matrix
```
Comment (ara4n) [inline gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx:15]:
```suggestion
The most glaring item on the list was the sluggish pace that many Matrix
```

File: gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+authors, and want to shed some light on the reasoning behind it, and what we
+plan to do to prevent leaving authors in the dark about why there may be no
+Spec Core Team activity on their proposal.
+
+
+## Proposal Triaging
+
+There are currently 136 open MSCs that have yet to undergo Final Comment
+Period (FCP), 75 of which are marked as proposal-in-review, and 20 that have
+a FCP proposed. This is a lot of outstanding ideas, features and maintenance
```
Comment (ara4n) [inline gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx:30]: We need to put this in context - how many have we merged?

File: gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+changes.
+
+The Spec Core Team itself is made up of 8 members, each of which have
+separate full-time jobs. 7 of these are employees of New Vector Ltd. All team
```
Comment (ara4n) [inline gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx:34]: The fact that 7/8 currently work for NV is irrelevant to this, imo.

File: gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+members are well-placed to be on the team given their wide breadth of
+knowledge across the Matrix.org ecosystem, however the majority are some of
+the most busy pushing forward Riot, Synapse and Dendrite. This limits the
```
Comment (ara4n) [inline gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx:37]:
```suggestion
the most busy pushing forward Matrix's reference implementations - without which, Matrix will unquestionably fail. This limits the
```

File: gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx [hunk 16]
```diff
@@ -0,0 +1,83 @@
...
+amount of MSCs that the team can effectively work on at a given time.
+
+The team understands that there are MSCs that provide incredibly useful
+features, such as
+[support for LaTeX inmessages](https://github.com/matrix-org/matrix-doc/pull/2191)
```
Comment (ara4n) [inline gatsby/content/blog/2020/04/2020-04-29-msc-categories.mdx:42]:
```suggestion
[support for LaTeX in messages](https://github.com/matrix-org/matrix-doc/pull/2191)
```

**Comment by ara4n** [comment]
lgtm mod tweaks.

**Pull request merged**